src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -466,6 +466,20 @@ case class TagFamilySizeMetric(family_size: Int,
                                var fraction: Proportion = 0,
                                var fraction_gt_or_eq_family_size: Proportion = 0) extends Metric
 
+/**
+ * Metrics produced by `GroupReadsByUmi` to describe reads passed through UMI grouping
+ * @param reads The number of reads accepted for grouping
+ * @param filteredNonPf The number of non-PF reads
+ * @param filteredPoorAlignment The number of templates discarded for poor
+ * @param filteredNsInUmi The number of templates discarded due to one or more Ns in the UMI
+ * @param filterUmisTooShort The number of templates discarded due to a shorter than expected UMI
```

Comment:
Some of the descriptions might need updating.

`filteredPoorAlignment` has a sentence fragment.

I also notice all the other metric descriptions end in periods, but these do not.

And are reads _**reads**_, or read pairs, or are they SAM records?

Reply:
They're SAM records. I was aligning with `logStats()`. If we're going to be explicit in the metrics file I think the logging should be updated as well.

Reply:
The way the tool works I think the context for this would mean `SAM records == reads`.

So it's probably OK to keep it as **_reads_** since that should be more intuitive for users.